browser/src/Services/BufferUpdates.ts

```diff
@@ -52,5 +52,14 @@ export class BufferUpdates {
                 this._pluginManager.notifyBufferUpdate(args, bufferLines)
             }
         })
+
+        this._neovimInstance.on("buffer-update-incremental", (args: Oni.EventContext, bufferLine: string, lineNumber: number) => {
+            this._lastArgs = args
+            // this._lastBufferLines = bufferLine
```

Comment:
I believe this needs to be set, similiar to how incremental update is handled above. I noticed the completion doesn't seem to be working correctly when typing in insert mode in TypeScript files - I think this may be the root cause. (For example, typing `window` on a new line behaves differently on this branch vs master).